tests/cfg.rs

```diff
@@ -229,6 +229,38 @@ fn works_through_the_registry() {
 }
 
 #[test]
+fn ignore_version_from_other_platform() {
+    let this_family = if cfg!(unix) {"unix"} else {"windows"};
```

Comment:
This unittest is based on the two tests above. Both of them include `if !is_nightly() { return }`. It seems to work fine on rust 1.11, so I did not include that here.